### DISCUSSION TITLE
Pushover integration -- SaaS vs self-hosted versions

### DISCUSSION THREAD

**Discussion by jl-678** [healthchecks/healthchecks#837] [Q&A] [answered]
I just got finished configuring the self-hosted version of Healthchecks and it is working great! Thank you. 

One curiosity is that it seems that integrations are handled differently in the two versions. Specifically, there are many more services in the "Add More" section of the SaaS product like for example Pushover. Pushover is not available under "Add More" in the self-hosted version but is available by adding Apprise which I have since enabled.

I am curious about the difference and if there is a setting somewhere to mimic the SaaS behavior on the self-hosted version.

TY!

**Comment by cuu508** [comment, marked as answer]
See [README](https://github.com/healthchecks/healthchecks#pushover) for enabling the Pushover integration.

The integration types you see by default are the ones that do not need any additional configuration. The missing ones require additional setup. See README for pointers, and [the list of env variables](https://healthchecks.io/docs/self_hosted_configuration/) to see what env variables you will need to set. 

All the integrations you see in the SaaS product can be set up when self-hosting (but it can take some work. Jumping through all the Twilio & Meta onboarding hoops to enable WhatsApp is a bit of a process, for example). 

The self-hosted instances can also use a couple of integrations that are not enabled on https://healthchecks.io: Apprise and Shell commands.
